src/components/gauge/gauge.tsx

```diff
@@ -139,6 +140,17 @@ export class Gauge implements GxComponent {
       );
     }
 
+    function renderRangeLabels({ name, amount, color }): HTMLElement {
```

Comment:
Similar to the previous comment, this function doesn't return an actual HTMLElement, but JSX (that's an object that represents an element, but not an actual DOM element).

Reply:
I understand the mistake in the concept. It isn't an HTML element from DOM, it's a JSX object that represents a future HTMLElement.
Fixed in 65fb97b